FreeTubeApp/FreeTube
# Subscriptions: Add community tab

# Subscriptions: Add community tab


## Pull Request Type

## Related issue
closes https://github.com/FreeTubeApp/FreeTube/issues/3593

## Description
If you have less than 125 subscriptions in your profile then a tab is added on the subscription page for community posts.

## Screenshots 

![image](https://github.com/FreeTubeApp/FreeTube/assets/78101139/2624a9fe-ec66-4183-abdb-5210a0ec2908)

## Testing 
(Local + Invidious)
- view subscription page (less than 125 subscriptions)
- see community tab
- click on community tab
- see posts ordered by date

- disable community tab is Distraction Free Settings
- view subscription page
- dont see community tab

- have over 125 subscriptions
- view subscription page
- dont see community tab

## Desktop
- **OS:** windows
- **OS Version:** 10
- **FreeTube version:** 0.19.0
